libpod/boltdb_state.go

```diff
@@ -274,6 +304,14 @@ func (s *BoltState) LookupContainer(idOrName string) (*Container, error) {
 			// Use else-if in case the name is set to a partial ID
 			exists := false
 			err = idBucket.ForEach(func(checkID, checkName []byte) error {
+				// If the container isn't in our namespace, we
```

Comment:
(It’s asymmetrical and a bit non-obvious that we don’t have to worry about excluding different-namespace containers in the exact match case, because `getContainerFromDB` will do that. Maybe worth a comment. Same in `LookupPod`.)

Reply:
Done